proto/routerpb.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package routerpb;
+
+import "pdpb.proto";
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unkeyed_all) = false;
+option (gogoproto.goproto_unrecognized_all) = false;
+option (gogoproto.goproto_sizecache_all) = false;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+service Router {
+    rpc GetRegion(pdpb.GetRegionRequest) returns (pdpb.GetRegionResponse) {}
+
+    rpc GetPrevRegion(pdpb.GetRegionRequest) returns (pdpb.GetRegionResponse) {}
+
+    rpc GetRegionByID(pdpb.GetRegionByIDRequest) returns (pdpb.GetRegionResponse) {}
+
+    rpc QueryRegion(stream pdpb.QueryRegionRequest) returns (stream pdpb.QueryRegionResponse) {}
+
+    rpc BatchScanRegions(pdpb.BatchScanRegionsRequest) returns (pdpb.BatchScanRegionsResponse) {}
```

Comment:
Do we need `ScanRegions`?

Reply:
ScanRegions is deprecated.

Reply:
Do br/lightning finish this adaptation?🤔

Reply:
No, but before we release this feature, we can check if the adaptation is finished again.